test/src/modules/navigation_tests.coffee

```diff
@@ -39,6 +39,14 @@ navigationTest "following a same-origin data-turbolinks=true link inside a data-
     assert.equal(navigation.action, "push")
     done()
 
+navigationTest "following a same-origin link with ID hash", (assert, session, done) ->
```

Comment:
Cool. Updated in https://github.com/turbolinks/turbolinks/pull/270/commits/2aafe10f754c5c81d071ad6fdea5be2f81376984 (I think I went round in circles: fragment identifier, anchor, hash, fragment identifier, anchor, hash … :)